### ISSUE TITLE
Adapt doc site from open ephys

### ISSUE THREAD

**Issue by matiasandina** [KravitzLabDevices/FED3#11]
The page is building but it's looking for an `index.html` on the repo that isn't being created or is not pushed to Github for some reason.

Anyhow, right now it's not rendering as expected and we get a sort of 404 error. I think some of the config files are using open ephys info. I think we are not seeing the `gh-pages` branch because this line is trying to commit to the open ephys repo instead of this repo.

https://github.com/KravitzLabDevices/FED3/blob/d8d771810eef658bfcd367d0fa3436c56eb85fe7/docs/.github/workflows/sphinx-build.yml#L44

I think there might be some token creation step that might also be missing:
See here where token is trying to be used to push to the `gh-pages` branch

https://github.com/KravitzLabDevices/FED3/blob/d8d771810eef658bfcd367d0fa3436c56eb85fe7/docs/.github/workflows/sphinx-build.yml#L52-L58

**Comment by ahleighton** [comment]
Hi! I took a look at the docs generation. A lot of the config was still set to Open Ephys repos etc., so I've adjusted that to FED3, and manually added a gh-pages branch with a nojekyll file; my [fork](https://github.com/ahleighton/FED3) now generates [the doc site](https://ahleighton.github.io/FED3/) as expected.

**Comment by matiasandina** [comment]
Great, thank you so much for doing this. @KravitzLab Do we want to proceed with migrating the docs there?

**Comment by KravitzLab** [comment]
Yes!  It would be great to start migrating the documentation from the FED3 Wiki there and figure out what holes we need to patch.  Although I have to admit I'm still confused about how to edit the new docs site and I don't have time to dig in right now!  Matias, if you have time to do anything that would be great.  I'll have time in ~1 month to help out too!
________________________________
